src/tasks/processBulkUpload.ts

```diff
@@ -312,12 +333,20 @@ export const processBulkUpload = async (
   helpers.logger.debug(`Started processBulkUpload Job for Bulk Upload ID ${payload.bulkUploadId}`);
   const bulkUpload = await loadBulkUpload(payload.bulkUploadId);
   let bulkUploadFile: FileResult;
+  let bulkUploadHasFailed = false;
   try {
     await updateBulkUploadStatus(bulkUpload.id, BulkUploadStatus.IN_PROGRESS);
     bulkUploadFile = await downloadS3ObjectToTemporaryStorage(
       bulkUpload.sourceKey,
       helpers.logger,
     );
+  } catch (error) {
```

Comment:
This PR was started before the recent updates to error logging; I'll go through and fix any instances of `error` that should be `err`